types/material-colors/material-colors-tests.ts

```diff
@@ -0,0 +1,7 @@
+import * as colors from 'material-colors';
```

Comment:
It might be better to use the `import package = require` syntax here, since it's a CommonJS module:

```suggestion
import colors = require('material-colors');
```